### PR TITLE
new feature TabbedGroupPicker

### PR DESCRIPTION
**Pull request by adrianossi**
## **Description**

Create the NovoTabbedGroupPicker element.

#### **Verify that...**

- [x] Any related demos were added and `npm start` and `npm run build` still works
- [x] New demos work in `Safari`, `Chrome` and `Firefox`
- [x] `npm run lint` passes
- [x] `npm test` passes and code coverage is increased
- [x] `npm run build` still works

#### **Bullhorn Internal Developers**
- [x] Run `Novo Automation`
- [x] Run `BBO Automation`

##### **Screenshots**

### CODE REVIEW

File: projects/novo-elements/src/elements/tabbed-group-picker/TabbedGroupPicker.ts [hunk 1]
```diff
@@ -73,6 +82,47 @@ export class NovoTabbedGroupPickerElement implements OnInit {
     });
   }
 
+  ngAfterViewInit(): void {
+    this.scrollableInstance = [...this.scrollDispatch.scrollContainers.keys()].find((scrollable) =>
```
Comment (DrewHoo) [inline projects/novo-elements/src/elements/tabbed-group-picker/TabbedGroupPicker.ts]: there is probably a smarter way to do this

**Comment by DrewHoo** [comment]
@aossi lmk if you want to get on a call once you get back in the office and have had a chance to look at this and the novo side. I don’t start my new job until tomorrow so I’ve got some time.

File: projects/novo-elements/src/elements/tabbed-group-picker/TabbedGroupPicker.ts [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+    this.displaySchemaIndex = this.schemata.map(({ typeName }) => typeName).indexOf(schema.typeName);
+  }
+
+  get viewportClass(): string {
+    return `virtual-scroll-viewport ${this.title}`;
+  }
+
+  get virtualScrollItemSize() {
+    const em = 3.1; // this corresponds to TabbedGroupPicker.scss .tabbed-group-picker-column-container.tabbed-group-picker-column &.right novo-list-item.height
+    const defaultFontSize = 16; // fingers crossed
```
Comment (monroepe) [inline projects/novo-elements/src/elements/tabbed-group-picker/TabbedGroupPicker.ts]: I don't like comments like this in the code base.

Comment (adrianossi) [reply projects/novo-elements/src/elements/tabbed-group-picker/TabbedGroupPicker.ts]: Yup, I'll clean it up.

**Pull request merged**